includes/class-ckwc-integration.php

```diff
@@ -247,8 +249,25 @@ public function admin_options() {
 			 * Settings.
 			 */
 			default:
-				// Load WooCommerce Integration's Settings screen.
-				parent::admin_options();
```

Comment:
I assume you meant to delete this?

Reply:
Yes - we load a slightly modified view with the applicable markup and settings table here:
https://github.com/ConvertKit/convertkit-woocommerce/blob/09f0058254ebce2f08307d8f04022b2bab425c92/includes/class-ckwc-integration.php#L251-L271